logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -206,4 +230,25 @@ class ProtoContentMapperImpl(
             idMapper.fromProtoModel(qualifiedConversationID)
         else null
     }
+
+    private companion object {
+        val UNICODE_SELECTORS = listOf(
+            '\uFE00',
+            '\uFE01',
+            '\uFE02',
+            '\uFE03',
+            '\uFE04',
+            '\uFE05',
+            '\uFE06',
+            '\uFE07',
+            '\uFE08',
+            '\uFE09',
+            '\uFE0A',
+            '\uFE0B',
+            '\uFE0C',
+            '\uFE0D',
+            '\uFE0E',
+            '\uFE0F',
```

Comment:
What's this used for? are we even referencing it anywhere in Kalium or AR?

Reply:
Nice spot.

Indeed, it is no longer being used. It is a left-over from before we decided to split reactions by commas.
We were trying to actually parse unicode until we realise it was just easier and more human-readable to do it like this.

I'll remove it.